### DISCUSSION TITLE
How do I define a cutting board recipe in a custom datapack?

### DISCUSSION THREAD

**Discussion by Anomalocaridid** [vectorwing/FarmersDelight#822] [Q&A] [answered]
I'm trying to make a datapack that adds compatibility between different mods in a custom modpack I use. Currently, I am trying to make recipes that convert citrine and topaz blocks from Spectrum into their respective shards (like what is currently done for vanilla minecraft's amethyst blocks).However, it seems like no matter what I do, it does not add the recipe.

I am using Minecraft 1.19.2 with Fabric 0.14.21 and Farmer's Delight 1.19.2-1.3.10.1
Here's what I currently have, located in data/farmersdelight/recipes/citrine_block.json in the datapack:

```json
{
  "type": "farmersdelight:cutting",
  "ingredients": [
    {
      "item": "spectrum:citrine_block"
    }
  ],
  "result": [
    {
      "count": 4,
      "item": "spectrum:citrine_shard"
    }
  ],
  "tool": {
    "type": "farmersdelight:tool_action",
    "action": "pickaxe_dig"
  }
}
```

**Comment by vectorwing** [comment, marked as answer]
To make a custom Cutting Board recipe, follow this wiki article:
https://github.com/vectorwing/FarmersDelight/wiki/Cutting-Board-Recipes

However, this repository is for the **Forge** version of Farmer's Delight. I'm not sure if there are any syntax changes in the Fabric port; if the guide doesn't work, please forward this issue to their board instead. 👍

**Reply by Anomalocaridid** [reply]
My bad, I was not aware the Fabric version was a separate project. That was the cause.
